modules/test/playwright/tests/object-web/objectClientExtensions.spec.ts

```diff
@@ -168,3 +198,82 @@ test('Can create, read, update, and delete object entries that use the client ex
 			.getByText(objectEntryUpdatedValue, {exact: true})
 	).toBeHidden();
 });
+
+test('Can trigger object validation as a client extension', async ({
+	editObjectValidationPage,
+	modalAddObjectValidationPage,
+	objectValidationsPage,
+	page,
+	viewObjectEntriesPage,
+}) => {
+	const [objectDefinition] = createdEntities.objectDefinitions;
+
+	const objectField = objectDefinition.objectFields.find(
+		({system}) => !system
+	);
+
+	await objectValidationsPage.goto(objectDefinition.label['en_US']);
+
+	await objectValidationsPage.addObjectValidationButton.click();
+
+	const objectValidationLabel = 'ClientExtensionValidation' + getRandomInt();
+
+	await modalAddObjectValidationPage.fillObjectValidationInputs(
+		objectValidationLabel,
+		'Liferay Sample Etc Spring Boot Spring Boot Object Validation Rule 1'
+	);
+
+	await page.getByText(objectValidationLabel).click();
+
+	await objectValidationsPage.activeValitionToggle.check();
+
+	await objectValidationsPage.errorMessageInput.fill(
+		'This entry is not possible.'
+	);
+
+	await editObjectValidationPage.saveObjectValidationButton.click();
+
+	await page.waitForEvent('domcontentloaded');
+
+	await viewObjectEntriesPage.goto(objectDefinition.id);
+
+	await viewObjectEntriesPage.clickAddObjectEntry(
+		objectDefinition.label['en_US']
+	);
+
+	await viewObjectEntriesPage.fillObjectEntry({
+		objectFieldBusinessType: objectField.businessType,
+		objectFieldLabel: objectField.label['en_US'],
+		objectFieldValue: 'Invalid Name',
+	});
+
+	await viewObjectEntriesPage.saveObjectEntryButton.click();
+
+	await waitForAlert(page, 'This entry is not possible.', {type: 'danger'});
+
+	await viewObjectEntriesPage.backButton.click();
+
+	await expect(
+		page.locator('.cell-name').getByText('Invalid Name')
+	).toBeHidden();
+
+	await viewObjectEntriesPage.clickAddObjectEntry(
+		objectDefinition.label['en_US']
+	);
+
+	await viewObjectEntriesPage.fillObjectEntry({
+		objectFieldBusinessType: objectField.businessType,
+		objectFieldLabel: objectField.label['en_US'],
+		objectFieldValue: 'Valied Name',
+	});
+
+	await viewObjectEntriesPage.saveObjectEntryButton.click();
+
+	await waitForAlert(page);
+
+	await viewObjectEntriesPage.backButton.click();
+
+	await expect(
+		page.locator('.cell-name').getByText('Valid Name')
+	).toBeHidden();
```

Comment:
Shouldn't this have a `.not`, like `not.toBeHidden`?

Reply:
I switched to `toBeVisible`, I really didn't notice this glitch, thanks!